ouroboros-consensus/src/Ouroboros/Consensus/Mempool/TxSeq.hs

```diff
@@ -31,7 +31,7 @@ newtype TicketNo = TicketNo Word64
 
 -- | The transaction ticket number from which our counter starts.
 zeroTicketNo :: TicketNo
-zeroTicketNo = TicketNo 1
+zeroTicketNo = TicketNo 0
```

Comment:
This and [this](https://github.com/input-output-hk/ouroboros-network/pull/707/files#diff-f48335b7550e9a7952d6501cd52a4c01R182) change are both required fixes such that the new tests pass. I apparently didn't properly address a bug which Duncan had brought up in https://github.com/input-output-hk/ouroboros-network/pull/678#discussion_r297869328

Reply:
It's also worth noting that I'll be adding some more tests related to the `TicketNo` counter implementation in a PR for #692.